triple-store/triple-store-impl-rdf4j/src/main/java/com/powsybl/triplestore/impl/rdf4j/TripleStoreRDF4J.java

```diff
@@ -216,12 +225,40 @@ private static void createStatements(RepositoryConnection cnx, String objType, P
         });
     }
 
-    private static void write(Model m, OutputStream out) {
+    private void write(Model model, OutputStream out) {
         try (PrintStream pout = new PrintStream(out)) {
-            RDFWriter w = new PowsyblWriter(pout);
-            w.getWriterConfig().set(BasicWriterSettings.PRETTY_PRINT, true);
-            Rio.write(m, w);
+            RDFWriter writer = new PowsyblWriter(pout);
+            writer.getWriterConfig().set(BasicWriterSettings.PRETTY_PRINT, true);
+            if (writeBySubject) {
+                rioWriteBySubject(model, writer);
+            } else {
+                Rio.write(model, writer);
+            }
+        }
+    }
+
+    private void rioWriteBySubject(Model model, RDFWriter writer) {
```

Comment:
What `rio` mean?

Reply:
stands for RDF Input/Output, will remove it from the method name, it is not relevant here